### PR TITLE
Update to RP standard release 3.5 (#29)

### PR DESCRIPTION
**Pull request by leonardbf**
Update of cli-extension to the RP standard V3.5
Includes export policy and active directory functionality
Rename of anf commands to netappfiles
(internal netapp ticket nfsaas-2102)

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by leonardbf** [comment]
A review is still required here. Thanks.

File: src/netappfiles-preview/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Release History
 ===============
 
+0.2.0
++++++
```
Comment (tjprescott) [inline src/netappfiles-preview/HISTORY.rst]: There should be something here.

Comment (leonardbf) [reply src/netappfiles-preview/HISTORY.rst]: I will add something in the next release which is coming in the next couple of weeks.

**Comment by leonardbf** [comment]
Can this be merged or is there something still required? Thanks.

**Comment by tjprescott** [comment]
@leonardbf it looks like you need to pull the latest changes from master, but otherwise this can be merged.

**Comment by leonardbf** [comment]
> @leonardbf it looks like you need to pull the latest changes from master, but otherwise this can be merged.

Ok, latest changes from master have now been merged to the fork.

**Pull request merged**